package.json

```diff
@@ -15,7 +15,9 @@
     "new:node": "nx generate @nx/node:application --directory=apps --docker",
     "new:lib": "nx g @nx/node:library --directory=libs",
     "docker-build:affected": "nx affected -t docker-build --prod --parallel --maxParallel=3 --skip-nx-cache --skip-nx-cache",
-    "compose:up": "yarn docker-build:affected && docker-compose up"
+    "compose:up": "yarn docker-build:affected && docker-compose up",
+    "swagger": "ts-node scripts/fetchCoingeckoSwagger",
+    "gen:types": "npx openapi-typescript"
```

Comment:
added postinstall

Reply:
That's fine by me. Just meant to run it automatically as we'll for sure forget it.
And it was removed because it depended on The Graph endpoint, which no longer exists.